server/service/hosts_test.go

```diff
@@ -446,11 +446,6 @@ func TestHostDetailsOSSettings(t *testing.T) {
 			switch c.host.Platform {
 			case "windows":
 				require.False(t, ds.GetHostMDMAppleProfilesFuncInvoked)
-				if c.licenseTier == fleet.TierPremium {
-					require.True(t, ds.GetHostMDMFuncInvoked)
-				} else {
-					require.False(t, ds.GetHostMDMFuncInvoked)
-				}
```

Comment:
seems like we're still calling this function in `svc.getHostDetails`, sanity check that this makes sense, either we're not calling it anymore in the test (might indicate a bug?) or we're calling it and we should keep the check 🤔

Reply:
Looks like you're right, mist have removed it by mistake 🙂